### PR TITLE
Build server: use App Store Connect API key

### PR DESCRIPTION
**Pull request by mcleinman**
**Fixes Phabricator ticket:** https://phabricator.wikimedia.org/T271350

### Notes
* The key is on the server - and is not committed to the repo.

### Test Steps
1. Using jenkins, build a build - using this branch. Ensure it's uploaded to app store connect.

### CODE REVIEW

File: fastlane/Fastfile [hunk 1]
```diff
@@ -272,6 +272,12 @@ platform :ios do
     ipa_path = "#{build_dir}/#{product_name}.ipa"
     build_number =  options[:build] || get_latest_build_number(prefix: "betas") + 1
     version = get_version_number_from_options(options)
+    apple_key = `cat ~/AuthKey_8L2XY35BLF.p8`
+    api_key = app_store_connect_api_key(
```
Comment (tonisevener) [inline fastlane/Fastfile:276]: Possibly overkill but I think we ought to try and mask the key and issuer ID as well, either via Jenkins environment variables or use a similar pattern as how you're pulling the auth key here.

**Comment by mcleinman** [comment]
Will fix this next week.

File: fastlane/Fastfile (hunk 1 above)
Comment (mcleinman) [reply fastlane/Fastfile:278]: At long last, removed. (Ran into some issues - the files can't have any newline whitespace at the end of the file, but it's working now.) 

I'm going to cycle the keys on the server, but I think this is ready to review again.

**Review by tonisevener — APPROVED** [review]
Works like a dream!

**Pull request merged**